README.md

```diff
@@ -76,17 +76,17 @@ You can override the global configuration by passing overrides to `:telemetry.at
 
 The following configuration options are supported:
 
-|Option|Description|Default|
-|-|-|-|
-|`tracer`|Tracer instance to use for reporting traces (*required*)||
-|`service`|Service name for Ecto traces|`ecto`|
-|`truncate`|Maximum length of a query (excess will be truncated)|5000|
+| Option     | Description                                              | Default |
+| ---------- | -------------------------------------------------------- | ------- |
+| `tracer`   | Tracer instance to use for reporting traces (_required_) |         |
+| `service`  | Service name for Ecto traces                             | `ecto`  |
+| `truncate` | Maximum length of a query (excess will be truncated)     | 5000    |
```

Comment:
Oops - some auto-formatting came with it. Want me to  revert that?

Reply:
I’d say leave it like this… I think it’s actually easier to read this way when looking at the markdown itself. 👍